### ISSUE TITLE
iOS Safari: Doesn't change value of <input>, shows console errors

### ISSUE THREAD

**Issue by alanbuchanan** [sampotts/rangetouch#6]
iOS 10.3
Safari

Given I import `rangetouch` using a `script` tag in the HTML page
And I interact with my `<input type="range>"
The `thumb` moves but the position doesn't update after releasing
And I get console errors

Without rangetouch:
![slider-without-rangetouch](https://user-images.githubusercontent.com/10364894/27392059-8137e1ec-569d-11e7-8ae9-6fdf3fbc3193.gif)

With rangetouch:
![slider-with-rangetouch](https://user-images.githubusercontent.com/10364894/27392068-85424a70-569d-11e7-9d61-bfd91f872ded.gif)

Console after interacting:
![screenshot 2017-06-21 at 16 15 35](https://user-images.githubusercontent.com/10364894/27392161-cc49eb4e-569d-11e7-8931-8c6dfc552916.png)

Error:
```
TypeError: e is not a function. (In 'e(e.target,e.type===l.events.end?"change":"input")', 'e' is an instance of TouchEvent)
c — rangetouch.js:1:1405
```

**Comment by sampotts** [comment]
Damn. Sorry about this. Looking now.

**Comment by sampotts** [comment]
Thanks for reporting this. Should be resolved in [v1.0.5](https://github.com/sampotts/rangetouch/releases/tag/v1.0.5)

**Comment by alanbuchanan** [comment]
Wow, thanks for the really fast fix.
